### PR TITLE
Secondary role traitor

### PR DESCRIPTION
**Pull request by Prakkkmak**

### CODE REVIEW

**Comment by Prakkkmak** [comment]
[Traitor skin => other role skin](https://trello.com/c/fSEG64hi/114-traitor-skin-other-role-skin)

File: code/Modules/Roles/RoleDistributor.cs [hunk 1]
```diff
@@ -12,7 +12,8 @@ public class RoleDistributor
     {
         private HashSet<MainPlayer> players;
         public Scenario Scenario;
-
+        public Role DefaultRole = new Assistant();
```
Comment (Nowp) [inline code/Modules/Roles/RoleDistributor.cs:15]: Default should be in Scenario

File: code/Modules/Roles/RoleDistributor.cs [hunk 2]
```diff
@@ -129,31 +125,41 @@ private IEnumerable<MainPlayer> GetPlayersWithRole()
...
+            return players.Where(mainPlayer => role.Equals(mainPlayer.RoleHandler.Role)).ToList();
         }
 
         /// <summary>
         /// Give a role to the player
         /// </summary>
         /// <param name="player">The player</param>
         /// <returns>If false, the player have the default assistant role</returns>
-        private bool GivePreferedRoles(MainPlayer player)
+        private bool GivePreferedRole(MainPlayer player)
```
Comment (Nowp) [inline code/Modules/Roles/RoleDistributor.cs:136]: GivePreferredRole*

File: code/Modules/Roles/RoleDistributor.cs (hunk 1 above)
Comment (Prakkkmak) [reply code/Modules/Roles/RoleDistributor.cs:15]: Yes but I don't want to change how the scenario is created for now. I put this here for reduce the dept. If you really want i can make the change

Comment (Nowp) [reply code/Modules/Roles/RoleDistributor.cs:15]: Nope it's fine but we'll change it in the future maybe

**Review by Nowp — APPROVED** [review]

**Pull request merged**